public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
This should be changed to reflect what our website is about. The information will be seen on google search results.

Reply:
Sorry for the late response. I will fix it. Thankss